wouso/games/teamquest/tests.py

```diff
@@ -200,6 +208,17 @@ def test_quest_status_create(self):
                 self.assertEqual(team_quest_question.level, level_status)
                 self.assertTrue(team_quest_question.question in level_status.level.questions.all())
 
+    def test_quest_status_progress(self):
```

Comment:
This only checks the progress when all questions are answered. You should rename this to test_quest_status_progress_100 and write one more test for partial progress.